src/joint.dia.graph.js

```diff
@@ -57,6 +43,8 @@ joint.dia.GraphCells = Backbone.Collection.extend({
 
 joint.dia.Graph = Backbone.Model.extend({
 
+    activeBatches: {},
```

Comment:
Can we rename this to `_batches`? I don't think we need to explicitly say that these are "active", and I want it to be clear that this is an internal variable.